src/deepsparse/server/system_logging.py

```diff
@@ -131,7 +136,7 @@ def log_resource_utilization(
 
 def log_request_details(
     server_logger: BaseLogger,
-    prefix: str,
+    prefix: str = REQUEST_DETAILS_IDENTIFIER_PREFIX,
```

Comment:
Do we want prefix only or should this be "group" and match if the value is *anywhere* in the identifier